style.css

```
@@ -1,12 +1,36 @@
-
 /* general gradio fixes */
 
+@media (prefers-color-scheme: light) {
+    ::-webkit-scrollbar {
+        width: 10px;
+   }
+    ::-webkit-scrollbar-thumb {
+        background-color: #ccc;
+        border-radius: 10px;
+   }
+    ::-webkit-scrollbar-track {
+        background-color: #f5f5f5;
+   }
+}
+@media (prefers-color-scheme: dark) {
+    ::-webkit-scrollbar {
+        width: 10px;
+   }
+    ::-webkit-scrollbar-thumb {
+        background-color: #ccc;
+        border-radius: 10px;
+   }
+    ::-webkit-scrollbar-track {
+        background-color: var(--neutral-900);
+   }
+}

```

```suggestion
::-webkit-scrollbar {
    width: 10px;
}

::-webkit-scrollbar-thumb {
    background-color: #ccc;
    border-radius: 10px;
}

@media (prefers-color-scheme: light) {
    ::-webkit-scrollbar-track {
        background-color: var(--neutral-100);
    }
}
@media (prefers-color-scheme: dark) {
    ::-webkit-scrollbar-track {
        background-color: var(--neutral-900);
    }
}
```
Made it less redundant.

---

Decided I like the look of no track better

---

The color preference is a browser setting, so if your is set on dark and you launch webui in light mode (or vice versa) the media query will not work as expected. 

You can do the light mode styling first and then use `body:has(.dark)::-webkit-scrollbar-track` to style the main scrollbar, and `.dark::-webkit-scrollbar-thumb` for the rest.